dll/cpl/sysdm/lang/zh-TW.rc

```diff
@@ -307,44 +312,44 @@ STRINGTABLE
 BEGIN
     IDS_CPLSYSTEMNAME "系統"
     IDS_CPLSYSTEMDESCRIPTION "查看您的電腦系統的信息並更改各種系統及硬體設定"
-    IDS_MEGABYTE "MB內存"
-    IDS_GIGABYTE "GB內存"
-    IDS_TERABYTE "TB內存"
-    IDS_PETABYTE "PB內存"
+    IDS_MEGABYTE "MB 記憶體"
+    IDS_GIGABYTE "GB 記憶體"
+    IDS_TERABYTE "TB 記憶體"
+    IDS_PETABYTE "PB 記憶體"
     IDS_MEGAHERTZ "MHz"
     IDS_GIGAHERTZ "GHz"
-    IDS_UPTIME_FORMAT "%u Days, %02u:%02u:%02u"
+    IDS_UPTIME_FORMAT "%u 天, %02u:%02u:%02u"
     IDS_VARIABLE "變量"
     IDS_VALUE "值"
     IDS_NO_DUMP "(無)"
-    IDS_MINI_DUMP "小內存轉儲 (64KB)"
-    IDS_KERNEL_DUMP "核心內存轉儲"
-    IDS_FULL_DUMP "完全內存轉儲"
+    IDS_MINI_DUMP "小記憶體傾印 (64KB)"
+    IDS_KERNEL_DUMP "核心記憶體傾印"
+    IDS_FULL_DUMP "完全記憶體傾印"
     IDS_USERPROFILE_NAME "名稱"
     IDS_USERPROFILE_SIZE "大小"
     IDS_USERPROFILE_TYPE "類型"
     IDS_USERPROFILE_STATUS "狀態"
     IDS_USERPROFILE_MODIFIED "修改"
-    IDS_USERPROFILE_LOCAL "Local"
-    IDS_USERPROFILE_ROAMING "Roaming"
-    IDS_USERPROFILE_MANDATORY "Mandatory"
-    IDS_USERPROFILE_CONFIRM_DELETE "Are you sure you want to delete %s's profile?"
-    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "Confirm Delete"
-    IDS_USERPROFILE_ACCOUNT_UNKNOWN "Account Unknown"
-    IDS_USERPROFILE_ACCOUNT_DELETED "Account Deleted"
-    IDS_USERPROFILE_TYPE_TEXT "When %s logs onto this computer, should the operating system use the roaming profile or just the locally cached copy of the roaming profile."
+    IDS_USERPROFILE_LOCAL "本機"
+    IDS_USERPROFILE_ROAMING "漫遊"
+    IDS_USERPROFILE_MANDATORY "強制"
+    IDS_USERPROFILE_CONFIRM_DELETE "您是否確定要刪除 %s 的設定檔?"
+    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "確認刪除"
+    IDS_USERPROFILE_ACCOUNT_UNKNOWN "不明的帳戶"
+    IDS_USERPROFILE_ACCOUNT_DELETED "帳戶已刪除"
+	IDS_USERPROFILE_TYPE_TEXT "當 %s 登入這台電腦時, 作業系統應使用漫遊設定檔，還是漫遊設定檔的本機快取複本。"
     IDS_MESSAGEBOXTITLE "系統控制面板小程序"
-    IDS_WARNINITIALSIZE "爲分頁文件的初始大小輸入一個數字值。"
-    IDS_WARNMAXIMUMSIZE "爲分頁文件的最大大小輸入一個數字值。"
+    IDS_WARNINITIALSIZE "為分頁文件的初始大小輸入一個數字值。"
+    IDS_WARNMAXIMUMSIZE "為分頁文件的最大大小輸入一個數字值。"
     IDS_WARNINITIALRANGE "分頁文件的初始大小不得小於 2 MB，並且不得超過所選磁碟上的可用空間。"
     IDS_WARNMAXIMUMRANGE "分頁文件的最大大小不能小於其初始大小和不能超過所選磁碟上的可用空間。"
     IDS_PAGEFILE_MB "%u MB"
     IDS_PAGEFILE_NONE "無"
     IDS_PAGEFILE_SYSTEM "系統已管理"
-    IDS_INFOREBOOT "您必須重新啓動電腦，才能套用更改。"
+    IDS_INFOREBOOT "您必須重新啟動電腦，才能套用更改。"
     IDS_HWPROFILE_CONFIRM_DELETE_TITLE "確認設定檔刪除"
-    IDS_HWPROFILE_CONFIRM_DELETE "你確定你想要刪除的硬件設定檔？ ""%s""?"
-    IDS_HWPROFILE_ALREADY_IN_USE "配置式名稱已在使用。"
+    IDS_HWPROFILE_CONFIRM_DELETE "你確定你想要刪除的硬件設定檔 ""%s""？"
```

Comment:
should we make it consistent to use Chinese question mark "？"or English one "?"

Reply:
yes indeed